frontend/tasks/dist.js

```diff
@@ -12,11 +12,13 @@ gulp.task('dist-rev-assets', function() {
       'favicon.ico',
       /static\/addon\/*/,
       /static\/locales\/*/,
+      /static\/images\/experiments\/*\/social\/*/,
```

Comment:
I'm not 100% sure I follow this (but that rarely stops me from making stupid, uninformed comments)...
I don't think this RegExp will work for all experiments, since some have absurdly absurd Django file structures still (notably "activity-stream" and "no-more-404s"; and "universal-search" is `null`, so I'm guessing those would revert back to defaults):

1. #### [/content-src/experiments/activity-stream.yaml](https://github.com/mozilla/testpilot/blob/master/content-src/experiments/activity-stream.yaml):

    ```yaml
    image_twitter: /static/images/experiments/activity-stream/experiments_experiment/e/1/e19739ea25188c866b654d3aa2616b27_image_1473186642_0579.png
    image_facebook: /static/images/experiments/activity-stream/experiments_experiment/e/1/e19739ea25188c866b654d3aa2616b27_image_1473186642_0826.png
    ```

1. #### [/content-src/experiments/cliqz.yaml](https://github.com/mozilla/testpilot/blob/master/content-src/experiments/cliqz.yaml):

    ```yaml
    image_twitter: /static/images/experiments/cliqz/social/cliqz-twitter.jpg
    image_facebook: /static/images/experiments/cliqz/social/cliqz-facebook.jpg
    ```

1. #### [/content-src/experiments/containers.yaml](https://github.com/mozilla/testpilot/blob/master/content-src/experiments/containers.yaml):

    ```yaml
    image_twitter: /static/images/experiments/containers/social/containers-twitter.jpg
    image_facebook: /static/images/experiments/containers/social/containers-facebook.jpg
    ```

1. #### [/content-src/experiments/min-vid.yaml](https://github.com/mozilla/testpilot/blob/master/content-src/experiments/min-vid.yaml):

    ```yaml
    image_twitter: /static/images/experiments/min-vid/social/mv-twitter.jpg
    image_facebook: /static/images/experiments/min-vid/social/mv-facebook.jpg
    ```

1. #### [/content-src/experiments/no-more-404s.yaml](https://github.com/mozilla/testpilot/blob/master/content-src/experiments/no-more-404s.yaml):

    ```yaml
    image_twitter: /static/images/experiments/no-more-404s/experiments_experiment/a/9/a9e6dc66c6a22851721715ae4d0641fb_image_1472573814_0582.png
    image_facebook: /static/images/experiments/no-more-404s/experiments_experiment/a/9/a9e6dc66c6a22851721715ae4d0641fb_image_1472573814_0278.png
    ```

1. #### [/content-src/experiments/page-shot.yaml](https://github.com/mozilla/testpilot/blob/master/content-src/experiments/page-shot.yaml):

    ```yaml
    image_twitter: /static/images/experiments/page-shot/social/ps-twitter.jpg
    image_facebook: /static/images/experiments/page-shot/social/ps-facebook.jpg
    ```

1. #### [/content-src/experiments/pulse.yaml](https://github.com/mozilla/testpilot/blob/master/content-src/experiments/pulse.yaml):

    ```yaml
    image_twitter: /static/images/experiments/pulse/social/pulse-twitter.jpg
    image_facebook: /static/images/experiments/pulse/social/pulse-facebook.jpg
    ```

1. #### [/content-src/experiments/snooze-tabs.yaml](https://github.com/mozilla/testpilot/blob/master/content-src/experiments/snooze-tabs.yaml):

    ```yaml
    image_twitter: /static/images/experiments/snooze-tabs/social/snooze-tabs-twitter.jpg
    image_facebook: /static/images/experiments/snooze-tabs/social/snooze-tabs-facebook.jpg
    ```

1. #### [/content-src/experiments/tab-center.yaml](https://github.com/mozilla/testpilot/blob/master/content-src/experiments/tab-center.yaml):

    ```yaml
    image_twitter: /static/images/experiments/tab-center/social/tc-twitter.jpg
    image_facebook: /static/images/experiments/tab-center/social/tc-facebook.jpg
    ```

1. #### [/content-src/experiments/tracking-protection.yaml](https://github.com/mozilla/testpilot/blob/master/content-src/experiments/tracking-protection.yaml):

    ```yaml
    image_twitter: /static/images/experiments/tracking-protection/social/tp-twitter.jpg
    image_facebook: /static/images/experiments/tracking-protection/social/tp-facebook.jpg
    ```

1. #### [/content-src/experiments/universal-search.yaml](https://github.com/mozilla/testpilot/blob/master/content-src/experiments/universal-search.yaml):

    ```yaml
    image_twitter: null
    image_facebook: null
    ```